app/routes.js

```diff
@@ -1,14 +1,36 @@
 const express = require('express')
 const router = express.Router()
+const userdata = require('./userdata')
 const jobRoleData = require('./jobRoleData.js')
+const userValidator = require('./validators/userValidator')
 
 router.get('/jobRoles', async (req, res) => {
-
     res.render('jobRolesView', {
       jobRoles: await jobRoleData.getJobRoles()
     }
     ); 
-  
-  });
+});
+
+router.post('/registeruser' , async (req, res) => {
+    let error = userValidator.validateUser(req.body)
+
+    if (error) {
+        res.locals.errormessage = error
+        return res.render('registration', req.body);
+    } else {
+        try {
+            var newUser = req.body;
+            await userdata.createUser(newUser);
```

Comment:
Could lines 22 and 23 be one line (CreateUser(req.body))

Reply:
Didn't spot that, cheers 😁